src/styles.css

```diff
@@ -71,14 +76,20 @@ div.srt-button {
   background-color: #112e51;
 }
 
+#art {
+background-color: #fa9441;
+outline: 0.25rem solid #2491ff;
+color: #1b1b1b;
+}
+
 .srt-button {
   background-color: #337ab7;
   border-radius: 5px;
   border: none #2e6da4;
   color: #fff;
   cursor: pointer;
   display: table;
-  float: right;
+  float: left;
```

Comment:
Are we positive that this doesn't affect the look of any other component in the UI?

Reply:
I double checked throughout the site and it seemed fine. Just in case, I just changed it back to float right, and added the float left to the css in results-detail.component.css art-button